source/DSCResources/DSC_CMSoftwareUpdatePoint/DSC_CMSoftwareUpdatePoint.schema.mof

```diff
@@ -0,0 +1,16 @@
+[ClassVersion("1.0.0"), FriendlyName("CMSoftwareUpdatePoint")]
+class DSC_CMSoftwareUpdatePoint: OMI_BaseResource
+{
+    [Key, Description("Specifies the SiteCode for the Configuration Manager site.")] String SiteCode;
+    [Required, Description("Specifies the Site Server to install or configure the role on.")] String SiteServerName;
```

Comment:
This should be a key, and would need to be changed in the Readme as well

Reply:
Changed to Key